2-mandatory/8-journey-planner.js

```diff
@@ -19,10 +19,16 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+const journeyPlanner = (locations, transportMode) => {
+  arr = [];
+  for (let location in locations) {
+    if (locations[location].includes(transportMode)) arr.push(location);
+  }
+  return arr;
```

Comment:
Nice code here Laleh! The only thing I would say is that by using `if` and then not using `{}` Or even putting `arr.push(location)` on the next line down is considered bad code! I would always suggest using `{}` after using `if` , or doing the following
```
  for (let location in locations) {
    if (locations[location].includes(transportMode)) 
         arr.push(location);
  }
  return arr;
```
Again, what you have done is not wrong, but I just want to ensure you do the best practice and become the best coder possible!